package.json

```diff
@@ -14,10 +14,10 @@
   },
   "license": "MIT",
   "dependencies": {
-    "jsdom": "^9.8.3"
+    "jsdom": "^12.2.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.16.0",
-    "babel-preset-es2015": "^6.16.0"
+    "babel-cli": "^6.26.0",
```

Comment:
We can use Babel 7 now.

Reply:
Should we just drop Babel, since we're writing it for Node.js (as per your description)?

Reply:
@bogas04 Please let me know

Reply:
Yeah sure, we can drop babel